### PR TITLE
Changes blob pulsing significantly plus a few more blob fixes

### PR DESCRIPTION
**Pull request by ChangelingRain**
Blobs now properly remember their overmind and get color from it predictably.
Fixes blob cores and nodes failing to give the reagent name when examined.
Probably fixes a few issues accidentally.

I'm not sure this is actually faster than the old Pulse but fuck it I'm doing it anyway.
Actually I'm not sure about speed for most of this if there's better ways to do any of this tell me

### CODE REVIEW

File: code/game/gamemodes/blob/overmind.dm [hunk 1]
```diff
@@ -90,7 +91,7 @@
 		return
 
 	var/message_a = say_quote(message, get_spans())
-	var/rendered = "<font color=\"#EE4000\"><i><span class='game say'>Blob Telepathy, <span class='name'>[name]([blob_reagent_datum.name])</span> <span class='message'>[message_a]</span></span></i></font>"
```
Comment (Anonus) [inline code/game/gamemodes/blob/overmind.dm]: Fun fact; 'game say' and 'message' don't appear to be actual span classes.
'name' is just a fancy way to say bold, too.

**Comment by Zilenan91** [comment]
It's probably fine if blobs pulse slightly faster than usual. We've had unusually high amounts of players lately.

File: code/game/gamemodes/blob/overmind.dm (hunk 1 above)
Comment (MrStonedOne) [inline code/game/gamemodes/blob/overmind.dm]: Game say is actually two classes, as classes are space delimited

Comment (Anonus) [inline code/game/gamemodes/blob/overmind.dm]: Well, neither of those are span classes either!

File: code/game/gamemodes/blob/blobs/blob_mobs.dm [hunk 2]
```diff
@@ -13,9 +13,10 @@
 	maxbodytemp = 360
 	var/mob/camera/blob/overmind = null
 
-/mob/living/simple_animal/hostile/blob/proc/adjustcolors(a_color)
-	if(a_color)
-		color = a_color
+/mob/living/simple_animal/hostile/blob/update_icons()
+	..()
```
Comment (phil235) [inline code/game/gamemodes/blob/blobs/blob_mobs.dm]: I'm pretty sure `..()` isn't needed here.

File: code/game/gamemodes/blob/blobs/core.dm [hunk 3]
```diff
@@ -17,25 +17,29 @@
...
 	overlays.Cut()
 	color = null
 	var/image/I = new('icons/mob/blob.dmi', "blob")
-	I.color = a_color
+	if(overmind)
+		I.color = overmind.blob_reagent_datum.color
 	overlays += I
 	var/image/C = new('icons/mob/blob.dmi', "blob_core_overlay")
 	overlays += C
+	return
```
Comment (phil235) [inline code/game/gamemodes/blob/blobs/core.dm]: this `return` is superfluous

**Comment by Razharas** [comment]
Ok

**Pull request merged**